frontend/src/scenes/session-recordings/playlist/playlistLogic.ts

```diff
@@ -43,4 +43,20 @@ export const playlistLogic = kea<playlistLogicType>([
             }
         },
     })),
+    actionToUrl(({ values }) => ({
+        setIsFiltersExpanded: () => {
+            if (values.isFiltersExpanded === false) {
+                const currentSearchParams = new URLSearchParams(window.location.search)
+                currentSearchParams.delete('showFilters')
+                currentSearchParams.delete('filtersTab')
+                window.history.replaceState(
+                    {},
+                    '',
+                    `${window.location.pathname}${
+                        currentSearchParams.toString() ? '?' + currentSearchParams.toString() : ''
+                    }`
+                )
```

Comment:
you're better using kea router here so more like
```suggestion
		return [
			router.values.location.pathname,
			{
	                    ...router.values.searchParams,
	                    showFilters: undefined,
	                    filtersTab: undefined,
	                },
	                router.values.hashParams,
		]
```